ArchiSteamFarm/Json/Steam.cs

```diff
@@ -171,6 +174,25 @@ public sealed class Asset {
 			}
 #pragma warning restore IDE0051
 
+#pragma warning disable IDE0051
+			[JsonProperty(PropertyName = "instanceid", Required = Required.DisallowNull)]
+			private string InstanceIDText {
+				set {
+					if (string.IsNullOrEmpty(value)) {
+						return;
```

Comment:
If you do not expect this to be null or empty, then you should raise null error here.

Reply:
It can be empty in some cases, that's because there is no error raised.

Reply:
Alright then.